<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer_login_signup</class>
 <widget class="QDialog" name="customer_login_signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Username: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_username">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_password">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Please enter password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Password: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="customer_login">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>No account? Join now!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="customer_signup">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>816</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>customer_login_signup</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
